package.json

```diff
@@ -8,9 +8,9 @@
   "scripts": {
     "bootstrap": "lerna bootstrap --concurrency 1 --npm-client=\"yarn\" --hoist && node ./scripts/hoist-internals.js",
     "bootstrap:docs": "cd docs && yarn install",
-    "bootstrap:react-native-vanilla": "lerna exec --scope react-native-vanilla -- yarn install",
-    "bootstrap:crna-kitchen-sink": "lerna exec --scope crna-kitchen-sink -- yarn install",
-    "bootstrap:test-cra": "npm run build-packs && lerna exec --scope test-cra -- yarn install",
+    "bootstrap:react-native-vanilla": "lerna exec --scope react-native-vanilla -- npm install",
+    "bootstrap:crna-kitchen-sink": "lerna exec --scope crna-kitchen-sink -- npm install",
+    "bootstrap:test-cra": "lerna exec --scope test-cra -- npm install",
```

Comment:
npm run build-packs is now manual step for dev & separate task in CI.

If you have any input on how to improve the dev experience I'm all ears.